src/cfml/system/endpoints/ForgeBox.cfc

```diff
@@ -181,6 +181,49 @@ component accessors="true" implements="IEndpointInteractive" singleton {
 		}
 	}	
 
+	/*
+	* Figures out what version of a package would be installed with a given semver range without actually going through the installation.
+	* @slug Slug of package
+	* @version Version range to satisfy
+	* @entryData Optional struct of entryData which skips the ForgeBox call.
+	*/
+	function findSatisfyingVersion( required string slug, required string version, struct entryData ) {
+		
+			// Use passed in entrydata, or go get it from ForgeBox.
+			var entryData = arguments.entryData ?: forgebox.getEntry( arguments.slug );
```

Comment:
That's fine.  The chunk of code was copied from another method that used `entryData` from the `local` scope, so I kept it in the same scope just to reduce the number of things I could screw up.l